pkg/online/online.go

```diff
@@ -188,3 +212,41 @@ func setAppInstallState(appID string, status string) error {
 
 	return nil
 }
+
+func readConfigValuesFromInClusterSecret() (string, error) {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		return "", errors.Wrap(err, "failed to get cluster config")
+	}
+
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to create clientset")
+	}
+
+	configValuesSecrets, err := clientset.CoreV1().Secrets(os.Getenv("POD_NAMESPACE")).List(metav1.ListOptions{
+		LabelSelector: "kots.io/automation=configvalues",
```

Comment:
is there any way we could require a static name instead of using (+deleting) the first one and leaving the rest?